### ISSUE TITLE
UTC Format sometimes not recognized on some calendars

### ISSUE THREAD

**Issue by globenewmedia** [markuspoerschke/iCal#500]
Using version 2.x I have seen that giving no timezone you create a date like this assuming calendars accept that as UTC

DTSTART:20230207T000000
DTEND:20230210T000000

but should it not be

DTSTART:20230207T000000Z
DTEND:20230210T000000Z

to be recognized as UTC time?

**Comment by jgulledge19** [comment]
@markuspoerschke  any review on this issue? I have set up an event and trying to get the time load correctly into Outlook and this seems be an issue for me as well.

**Comment by mipapo** [comment]
Yep, I would also say that this is wrong - Don't get the correct one on Google calender too without "Z"

**Comment by richardhj** [comment]
I work with UTC in the underlying database but set the TimeZone to Europe/Berlin (primary region of users). This works for Apple, but not all clients (I guess Outlook?).

Has someone found a proper way of handling this?
Modifiyng the DateTime object to Europe/Berlin and $useTimezone=false?

```php
// WORKING ON APPLE CALENDAR BUT NOT ALL CLIENTS
$tz = new \DateTimeZone('Europe/Berlin');

(new Event())
  ->setOccurrence(new TimeSpan(
      // Setting the timezone will not change the underlying point-in-time.
      // But showing "UTC" to the user might be confusing.
      new DateTime($date->setTimezone($tz), true),
      new DateTime($date->setTimezone($tz), true)
  ))
;
```
